src/ucs/memory/memory_type.h

```diff
@@ -20,14 +20,16 @@ BEGIN_C_DECLS
      UCS_BIT(UCS_MEMORY_TYPE_ROCM_MANAGED))
 
 
-/*
```

Comment:
Since you include this in the main doc you have to document in doxygen style everything in the doc, including UCS_MEMORY_TYPES_CPU_ACCESSIBLE. If some of the macros or structures are not used directly by users it is better to exclude those from the document.
```
/** An interface */
class Intf
{
  public:
    /** A method */
    virtual void func() = 0;

    /// @cond TEST

    /** A method used for testing */
    virtual void test() = 0;

    /// @endcond
};

/// @cond DEV
/*
 *  The implementation of the interface
 */
class Implementation : public Intf
{
  public:
    void func();

    /// @cond TEST
    void test();
    /// @endcond

    /// @cond
    /** This method is obsolete and does
     *  not show up in the documentation.
     */
    void obsolete();
    /// @endcond
};

/// @endcond
```